nessie/jobs/create_edl_schema.py

```diff
@@ -52,7 +52,7 @@ class CreateEdlSchema(BackgroundJob):
 
     def run(self):
         app.logger.info('Starting EDL schema creation job...')
-        self.create_schema()
+        # self.create_schema()
```

Comment:
@lyttam Is this comment-out a leftover from testing?

Reply:
Whoops, yes! Will amend.